static/app/views/performance/transactionDetails/quickTraceMeta.spec.jsx

```diff
@@ -195,12 +171,9 @@ describe('QuickTraceMeta', function () {
         errorDest="issue"
         transactionDest="performance"
       />,
-      routerContext
+      {context: routerContext}
```

Comment:
do we need to pass the context here?

Reply:
Seems to work without the context, I'll remove it!